traffic_ops/v4-client/server_server_capabilities.go

```diff
@@ -16,17 +16,20 @@ package client
 */
 
 import (
+	"fmt"
 	"net/url"
 	"strconv"
 
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/traffic_ops/toclientlib"
 )
 
-// apiServerServerCapabilities is the API version-relative path to the
-// /server_server_capabilities API endpoint.
+// apiServerServerCapabilities is the API version-relative path to the /server_server_capabilities API endpoint.
```

Comment:
Why'd you modify this comment?

Reply:
To keep it on the same line.

Reply:
I had that line broken before 80 characters to accommodate the standard 80-column-width VT. Our project doesn't have a style guide rule for how long a comment line can/must be in a Go source file, so you're free to write your own comments as you please in that regard, but the 2-line nature of this comment in particular was very much intentional.

Reply:
ok, reverting back your comment.